build.gradle

```diff
@@ -93,6 +93,7 @@ examplesFolder.eachFile { File file ->
             }
 
             zipExamples.dependsOn it
+            it.dependsOn 'PatchExamples'
```

Comment:
You don't need the quotes around PatchExamples

Reply:
Done